app/assets/stylesheets/layouts/application.scss

```diff
@@ -1,10 +1,10 @@
 .hands-bg {
-  background: linear-gradient(to bottom, transparent, #0000004a), no-repeat, url(BackgroundImage.jpeg);
+  background: linear-gradient(to bottom, transparent, rgba(0, 0, 0, 0.29)), no-repeat, url(BackgroundImage.jpg);
   background-position-y: center;
   -webkit-background-size: cover;
   -moz-background-size: cover;
   -o-background-size: cover;
   top: 0;
   bottom: 0;
-  min-height: 100vh;
+  height: 111vh;
```

Comment:
??? why is this bleeding out to be 110% of browser's view height??

Also we should refrain from using an arbitrary number just to satisfy the condition that you are in at the moment, unless there is a good reason to do so.

Reply:
@sanghun89 do you have a suggestion?

Reply:
I would have to understand what you're trying to do here.

If you are doing that for resizing the background image, i suggest you mess with the background image attributes instead of the actual container

Reply:
@sanghun89 @gcwokeup I took a different approach and made a fixed image instead of background. Take a look and let me if that is cleaner/looks better
![image](https://user-images.githubusercontent.com/7453522/58138152-b0d06a80-7bfa-11e9-939f-aacaf2309f59.png)
![image](https://user-images.githubusercontent.com/7453522/58138166-bcbc2c80-7bfa-11e9-9aef-65956b521d70.png)

Reply:
So that does make it cleaner visually. Only caveat is the html template and accessibility. You don't necessarily want an `<img />` tag to be something other than a logo or diagram that plays along with the content. This is a similar case to using `<br />` tags to do margins.

It's not recommended, but it is better than having a content to have an arbitrary percentage of 111% of viewheight.

We can leave it as is, or try something like this?? I didn't test it
```suggestion
 /* as div */
  &__background {
    position: fixed;
    top: 0;
    left: 0;
    background-image: linear-gradient(to bottom, transparent, rgba(0, 0, 0, 0.29)), no-repeat, url(BackgroundImage.jpg);
     /* Preserve aspet ratio */
    background-size: 1200px auto;
    background-position: center center;
    width: 100vw;
    height: 100vh;
  }
```

Reply:
Yeah that actually worked pretty well and good point on accessibility @sanghun89. Did a little tweaking to get the background to fit right, but this should be it.